packages/electrode-react-webapp/lib/default-handlers.js

```diff
@@ -63,9 +63,19 @@ module.exports = function setup(options) {
       const manifest = bundleManifest();
       const manifestLink = manifest ? `<link rel="manifest" href="${manifest}" />\n` : "";
       const css = WEBPACK_DEV
-        ? data.devCSSBundle
-        : (data.cssChunk && `${prodBundleBase}${data.cssChunk.name}`) || "";
-      const cssLink = css && !data.criticalCSS ? `<link rel="stylesheet" href="${css}" />` : "";
+        ? (!Array.isArray(data.devCSSBundle) && [data.devCSSBundle]) || data.devCSSBundle
+        : data.cssChunk
+          ? (!Array.isArray(data.cssChunk) && [data.cssChunk]) || data.cssChunk
+          : [""];
```

Comment:
This nested ternary isn't pretty, but it seems like you strong prefer `const`, and without, it would mean a `let`.